collections/ansible_collections/cloudkit/config_as_code/roles/load_extra_vars/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Create temporary file
+  ansible.builtin.tempfile:
+    state: file
+    suffix: subscription
+  register: extra_vars
+
+- name: Render extra vars template
+  ansible.builtin.template:
+    src: "{{ load_extra_vars_file_path }}"
+    dest: "{{ extra_vars.path }}"
+    mode: "0644"
+
+- name: Load extra vars
+  ansible.builtin.include_vars:
+    file: "{{ extra_vars.path }}"
```

Comment:
Why do we do this instead of simply:

```
- name: Load extra vars
  ansible.builtin.include_vars:
    file: "{{ load_extra_vars_file_path }}"
```

?

Reply:
`ansible.builtin.include_vars` alone does not render jinja templates (e.g. : the `{{ lookup('env', 'AAP_PASSWORD') }}`).